pkg/util/encoding/encoding.go

```diff
@@ -913,18 +913,7 @@ func prettyPrintInvertedIndexKey(b []byte) (string, []byte, error) {
 // modified if the input string is expected to be used again - doing so could
 // violate Go semantics.
 func UnsafeConvertStringToBytes(s string) []byte {
-	if len(s) == 0 {
```

Comment:
If the go docs are to be believed I think we need to leave in the zero length check.  https://pkg.go.dev/unsafe#StringData

Reply:
Ok, added this back.